k8s/namespaces/ia/ia-case-integration-api/ia-case-integration-api.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
We need ingressHost: immigration-appeal-int.demo.platform.hmcts.net as part of demo yaml.

Also can we change the name as ia-case-int-api